### PR TITLE
Fix to pyOpenSSL stubs to pass stubtest

### PR DESCRIPTION
**Pull request by jolaf**

### CODE REVIEW

File: stubs/pyOpenSSL/@tests/stubtest_allowlist.txt [hunk 1]
```diff
@@ -1,4 +0,0 @@
-OpenSSL.SSL.Connection.__init__
-OpenSSL.SSL.Context.load_verify_locations
-OpenSSL.SSL.Context.set_verify
-OpenSSL.crypto
```
Comment (Akuli) [inline stubs/pyOpenSSL/@tests/stubtest_allowlist.txt:4]: stubtest complains that `OpenSSL.crypto` doesn't exist. If you don't want to figure out what's going on with that, you can leave `OpenSSL.crypto` in the allowlist for this PR.

**Review by srittau — APPROVED** [review]
Looks good, be we need to re-add OpenSSL.crypto back per Akuli's comment.

**Comment by jolaf** [comment]
> Looks good, be we need to re-add OpenSSL.crypto back per Akuli's comment.
I just wonder, why stubtest complains. `OpenSSL.crypto` definitely exists.

Here's the error output:
```
error: OpenSSL.crypto is not a module
Stub: at line 1
MypyFile:1(
  /home/runner/work/typeshed/typeshed/stubs/pyOpenSSL/OpenSSL/crypto.pyi)
Runtime:
<cryptography.utils._ModuleWithDeprecations object at 0x7f0217cf0b80>
```

The root of the problem seems to be here:
https://github.com/python/mypy/blob/master/mypy/stubtest.py#L206

Should that place in `stubtest` be fixed instead of hiding the error by excluding the test?

**Comment by jolaf** [comment]
> Fixing this would be great, of course.

I've just proposed a workaround to `mypy`.

File: stubs/pyOpenSSL/@tests/stubtest_allowlist.txt (hunk 1 above)
Comment (jolaf) [reply stubs/pyOpenSSL/@tests/stubtest_allowlist.txt:4]: It seems I've figured out what's going on and proposed a fix:
https://github.com/python/mypy/pull/10663

**Comment by srittau** [comment]
Per the feedback on the mypy bug, let's just add the item back in the allowlist.

**Pull request merged**